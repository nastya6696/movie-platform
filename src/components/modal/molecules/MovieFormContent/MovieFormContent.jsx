import styles from './styles.module.scss';

export const MovieFormContent = ({children}) => (
  <div className={styles.wrapper}>
    {children}
  </div>
)
