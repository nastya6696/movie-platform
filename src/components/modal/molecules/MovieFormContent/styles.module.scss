@import "src/assets/scss/common";

.wrapper {
  padding: 63px 60px 60px;
}
