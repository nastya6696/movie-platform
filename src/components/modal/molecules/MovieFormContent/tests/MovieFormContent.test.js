import ShallowRenderer from "react-test-renderer/shallow";
import React from "react";
import {MovieFormContent} from "../MovieFormContent";

test('should create simple MovieFormContent component', () => {
  const component = new ShallowRenderer().render(
    <MovieFormContent>
      any children
    </MovieFormContent>
  );
  expect(component).toMatchSnapshot();
})
