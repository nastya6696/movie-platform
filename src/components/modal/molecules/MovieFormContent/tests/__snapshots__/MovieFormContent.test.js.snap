// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create simple MovieFormContent component 1`] = `
<div
  className="wrapper"
>
  any children
</div>
`;
